code/drasil-website/lib/Drasil/Website/Body.hs

```diff
@@ -94,9 +93,9 @@ symbMap :: FolderLocation -> ChunkDB
 symbMap fl = cdb ([] :: [DefinedQuantityDict]) (map nw [webName, phsChgMtrl, twoD] ++
   map getSysName allExampleSI ++ map nw [pendulum, motion, rigidBody, blast,
   heatTrans, sWHT, water, pidC, target, projectile, crtSlpSrf, shearForce,
-  normForce, slpSrf] ++ [nw $ fctSfty ^. defLhs] ++ [game, physics, condition, glaSlab, intrslce,
-  slope, safety, factor]) ([] :: [ConceptChunk]) ([] :: [UnitDefn]) [] [] [] []
```

Comment:
These chunks need to be removed from the `ChunkDB` or else the website will crash with a `duplicate chunk found in mass insertion` error (for each of them individually).